src/app/auth/auth.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  constructor() { }
+
+  private authorization = false;
+
+  login() {
+    this.authorization = true;
+  }
+
+  logout() {
+    this.authorization = false;
+    localStorage.clear();
+  }
+
+  loggedIn(): boolean {
+    return !!localStorage.getItem('token');
```

Comment:
!! is not considered a best practise, please use a better approach of converting to boolean